Detectors/MUON/MCH/Calibration/src/MCHChannelCalibratorAlt.cxx

```diff
@@ -0,0 +1,131 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MCHCalibration/MCHChannelCalibrator.h"
+#include "Framework/Logger.h"
+#include "MathUtils/fit.h"
+#include "CommonUtils/MemFileHelper.h"
+#include "CCDB/CcdbApi.h"
+#include "DetectorsCalibration/Utils.h"
+#include <cassert>
+#include <iostream>
+#include <sstream>
+#include <TStopwatch.h>
+
+namespace o2
+{
+namespace mch
+{
+namespace calibration
+{
+
+using Slot = o2::calibration::TimeSlot<o2::mch::calibration::MCHChannelData>;
+using clbUtils = o2::calibration::Utils;
+
+//_____________________________________________
+void MCHChannelData::fill(const gsl::span<const o2::mch::calibration::PedestalDigit> digits)
+{
+  bool mDebug = true;
+  LOG(INFO) << "[MCHChannelData::fill] digits size " << digits.size();
+
+  for (auto& d: digits) {
+    auto solarId = d.getSolarId();
+    auto dsId = d.getDsId();
+    auto channel = d.getChannel();
+
+    size_t hIndex = solarId * 40 * 64 + dsId * 64 + channel;
+
+    for (size_t i = 0; i < d.nofSamples(); i++) {
+      const int s = d.getSample(i);
+      float w = 1;
+
+      mHisto[hIndex](s);
+
+      /*mEntries[solarId][dsId][channel] += 1;
```

Comment:
if not needed, better remoce the commented code

Reply:
See above